src/main/resources/org/biouno/unochoice/CascadeChoiceParameter/config.jelly

```diff
@@ -55,4 +55,7 @@
   <f:entry title="${%Enable filters}" field="filterable" help="${rootURL}/../plugin/uno-choice/help-filterableParameters.html">
     <f:checkbox name="parameter.filterable" checked="${instance.filterable}">${%Filterable}</f:checkbox>
   </f:entry>
+  <f:entry title="${%Start filtered length}" field="filterLength" help="${rootURL}/../plugin/uno-choice/help-filterLengthParameters.html">
```

Comment:
We may need to think a better description, though I don't have one. @imoutsatsos ? Maybe something like "Letters needed to active filtering". But we can do that after this gets merged, so nothing to change here @klimas7